rules-tests/Privatization/Rector/Property/PrivatizeFinalClassPropertyRector/Fixture/keep_used_protected.php.inc

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace Rector\Tests\Privatization\Rector\Property\PrivatizeFinalClassPropertyRector\Fixture;
+
+abstract class AbstractKeepUsedProtected {
+    protected $abstractOptions = [
+        'options' => [],
+    ];
+
+    public function __construct()
+    {
+        if (isset($this->options)) {
+            $this->abstractOptions['options'] = $this->options;
+        }
+    }
+}
+
+namespace Rector\Tests\Privatization\Rector\Property\PrivatizeFinalClassPropertyRector\Fixture;
```

Comment:
There should be only one namespace in the file.

Reply:
I missed that, will fix. This is produced by the button Create a Test on https://getrector.org/demo.